code/modules/clothing/head/misc.dm

```diff
@@ -496,6 +496,12 @@
 	icon_state = "duncecap"
 	item_state = "duncecap"
 
+/obj/item/clothing/head/party_hat
+	name = "party hat"
+	desc = "A party cone. Not useful in disguise as a unicorn or impaling foes."
+	icno_state = "birthdayhat"
```

Comment:
I'm a god, how can you kill a god? What a grand and intoxicating innocence. How could you be so naive? There is no escape. No recall or intervention can work in this place. Come, lay down your weapons, it is not too late for my mercy.